stubs/CoreGenericFunctions.phpstub

```diff
@@ -319,40 +312,40 @@ function array_fill_keys(array $keys, $value): array
  * @psalm-template TKey
  *
  * @param string $pattern
- * @param array<TKey,string> $input
+ * @param array<TKey,string> $array
  * @param 0|1 $flags 1=PREG_GREP_INVERT
  * @return array<TKey,string>
  */
-function preg_grep($pattern, array $input, $flags = 0)
+function preg_grep($pattern, array $array, $flags = 0)
 {
 }
 
 /**
- * @param resource $handle
+ * @param resource $stream
  * @param-out closed-resource $handle
  */
-function fclose(&$handle) : bool
+function fclose(&$stream) : bool
 {
 }
 
 /**
- * @param string $reference
+ * @param string $string
  * @param-out null $reference
```

Comment:
you should change $reference here too. There's probably a few phpdoc tags that possibly contains the param name that should be changed too